powerdns/tests/test_ownership.py

```diff
@@ -0,0 +1,88 @@
+"""Tests for record/domain ownership"""
+
+import base64
+
+from django.contrib.auth.models import User
+from django.core import mail
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+from powerdns.tests.utils import DomainFactory
+
+
+class TestOwnershipBase(TestCase):
+    """Base test class creating some users."""
+
+    def setUp(self):
+        self.user1 = User.objects.create_superuser(
```

Comment:
It doesn't work for some reason. If I use a factory, I cannot login. I don't know why and I don't think there is point debugging this.